pysat/utils/coords.py

```diff
@@ -89,6 +90,9 @@ def calc_solar_local_time(inst, lon_name=None, slt_name='slt'):
         name of the longtiude data key (assumes data are in degrees)
     slt_name : string
         name of the output solar local time data key (default='slt')
+    apply_modulus : bool
+        If True, slt values are confined to [0, 24) via a modulus operation.
+        If False, no modulus applied. (default=True)
```

Comment:
```suggestion
        If True, SLT values are confined to [0, 24), if False they may be positive or negative based on the value of their universal time relative to that of the first point included in the Instrument object. (default=True)
```

With this, it seems like there should also be an option to supply a reference time that defaults to the first time value.